staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -989,6 +989,24 @@ type KubeletConfiguration struct {
 	// +featureGate=GracefulNodeShutdown
 	// +optional
 	ShutdownGracePeriodCriticalPods metav1.Duration `json:"shutdownGracePeriodCriticalPods,omitempty"`
+	// ShutdownGracePeriodByPodPriority specifies the shutdown grace period for Pods based on their associated priority class value. Requires the GracefulNodeShutdown feature gate to be enabled.
+	// Each entry in the array represents the graceful shutdown time a pod with a priority class value that
+	// lies in the range of that value and the next higher entry in the list when the node is shutting down.
+	// Requires the GracefulNodeShutdown feature gate to be enabled.
+	// For example:
+	// |Pod priority class value|Shutdown period|
+	// |------------------------|---------------|
+	// | 100000                 |10 seconds     |
+	// | 10000                  |180 seconds    |
+	// | 1000                   |120 seconds    |
+	// | 0                      |60 seconds     |
```

Comment:
This seems more appropriate for website documentation on how this will work rather than trying to inline a table which may not render correctly in non-fixed width/whitespace sensitive fonts. Is there a shorter way we could document this?

Reply:
https://github.com/kubernetes/kubernetes/pull/102915#discussion_r666568211
Yes, I also think it's a bit long, if there is a better short description, I am very willing to update. 
Maybe we can remove the example

Reply:
Yeah I think we should go ahead and remove the example, as it is illustrative but doesn't actually match what a user would set in a config.